src/components/actions/previewActionsRoute.ts

```diff
@@ -57,6 +57,7 @@ export const createDisplayData = (actionsOutput: ActionResult[]): ActionResultDi
             result.output?.document?.filename ??
             'Generated Document',
           fileId: result.output?.document?.uniqueId,
+          filename: result.output?.document?.filename,
```

Comment:
Is this from another PR ?

Reply:
Yeah, forgot to mention this, forgot it was in here. It's just for the front-end "Document Preview" modal, we need this additional field, but didn't justify a PR of its own.